content/en/docs/Installation/linux.md

```diff
@@ -13,11 +13,130 @@ aliases:
 
-### Update and Install Prerequisites
+## Install Navidrome Using Pre-built Binary
 
-Ensure your system is up to date and install [ffmpeg](https://ffmpeg.org/download.html).
+To install Navidrome on a Linux system using a .deb file, you can follow a streamlined process that leverages the convenience of Debian package management. This method simplifies the installation by eliminating the need to manually download and extract binaries.
+
+Before you begin, ensure that your system is up to date and that you have ffmpeg installed, as it is a requirement for Navidrome to function properly.
+
+~~~bash
+sudo apt update
+sudo apt upgrade
+~~~
+
+### Download the .deb File
+
+1. **Visit the Navidrome Releases Page**: Go to the [Navidrome releases page](https://github.com/navidrome/navidrome/releases) on GitHub to find the latest .deb package suitable for your system architecture (e.g., amd64 for 64-bit systems).
+
+2. **Download the .deb File**: Use wget or your browser to download the .deb file. Replace navidrome_0.XX.X_amd64.deb with the actual file name from the releases page.
+
+~~~bash
+wget https://github.com/navidrome/navidrome/releases/download/v0.XX.X/navidrome_0.XX.X_amd64.deb
+~~~
+
+### Install and Configure
+
+There are two ways to install the package, `apt` and `dpkg`. `apt` is the usual method because it will automatically determine dependancies and install them (ffmpeg).
+
+Using `apt`:
+
+~~~bash
+apt install ./navidrome_0.XX.X_linux_amd64.deb
+~~~
+
+Using `dpkg`:
+
+Install the package and then resolve the dependancies:
+
+~~~bash
+sudo dpkg -i ./navidrome_0.XX.X_amd64.deb
+sudo apt install -f
+~~~
+
+**Configuration File**: After installation, Navidrome MUST be configured to run. The default path for the configuration file is /var/lib/navidrome/navidrome.toml. Create and edit the file using nano directly.
+
+~~~bash
+sudo nano /var/lib/navidrome/navidrome.toml
+~~~
+
+Add the following line to specify your music library path:
+
+~~~conf
+MusicFolder = "/path/to/your/music/library"
+~~~
+
+For additional configuration options see the [configuration options page](https://www.navidrome.org/docs/usage/configuration-options/).
+
+**Start the Navidrome Service**: Use systemctl to start the Navidrome service.
+
+~~~bash
+sudo systemctl start navidrome
+~~~
+
+**Enable Navidrome to Start on Boot**: To make Navidrome start automatically on system boot, enable the service.
+
+~~~bash
+sudo systemctl enable navidrome
+~~~
+
+**Check Service Status**: Verify that Navidrome is running correctly.
+
+~~~bash
+sudo systemctl status navidrome
+~~~
+
+If everything is set up correctly, Navidrome will be accessible via web browser: http://localhost:4533.
+
```

Comment:
Note that the default music path is `/opt/navidrome/music`, and that it is running as user `navidrome`